### DISCUSSION TITLE
Add battery icon on taskbar

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#225] [Feature Requests]
I think that a battery icon or indicator would be nice for laptop users using this web desktop in fullscreen because, in fullscreen mode, you cannot see your taskbar.

**Comment by DustinBrett** [comment]
Could you clarify what you mean by "in fullscreen mode, you cannot see your taskbar."? You mean the top taskbar of your phone? Or does daedalOS's taskbar vanish? Battery indicator is a good idea, and good use case for making a system tray. I will add that to my list now. Thanks.

**Reply by CorruptionVR** [reply]
when you right click on the taskbar and press "Enter full screen" or press f11. and im talking about the desktop operating systems taskbar, but that could apply too.

**Reply by DustinBrett** [reply]
What does this have to do with "you cannot see your taskbar"? Which taskbar and where?

**Reply by CorruptionVR** [reply]
like, window's taskbar

**Reply by DustinBrett** [reply]
Ah ok, I think I see what you are saying. So when you go into fullscreen and the native taskbar goes away, having the battery helps indicate.

**Reply by CorruptionVR** [reply]
yes
